src/_pytest/python.py

```diff
@@ -538,15 +537,10 @@ def _importtestmodule(self):
 
 
 class Package(Module):
-    def __init__(self, fspath, parent=None, config=None, session=None, nodeid=None):
```

Comment:
Is removing these parameters a breaking change, or is this ctor considered private already?

Reply:
Might be a breaking change.
@RonnyPfannschmidt ?

Reply:
Kept it.  (note to self: old HEAD: fca99a1)